src/store/RelayRecordWriter.js

```diff
@@ -94,7 +94,7 @@ class RelayRecordWriter {
    */
   getDataID(
     storageKey: string,
-    identifyingArgValue: ?string
+    identifyingArgValue: any
```

Comment:
Or maybe just use a `return Object.prototype.toString.call(identifyingArgValue)`;
i should search if flow has some type cast function.